augur/__init__.py

```diff
@@ -15,9 +15,8 @@
 from .io.print import print_err
 from .argparse_ import add_command_subparsers, add_default_command
 
-recursion_limit = os.environ.get("AUGUR_RECURSION_LIMIT")
-if recursion_limit:
-    sys.setrecursionlimit(int(recursion_limit))
+DEFAULT_AUGUR_RECURSION_LIMIT = 10000
+sys.setrecursionlimit(int(os.environ.get("AUGUR_RECURSION_LIMIT", DEFAULT_AUGUR_RECURSION_LIMIT)))
```

Comment:
It's a small thing, but I think important to test the value of the env var, e.g. as `if recursion_limit` did.

`AUGUR_RECURSION_LIMIT` may be (intentionally or unintentionally) defined in the environment but have no value (e.g. be the empty string). We wouldn't want to call `int('')`. With `.get("AUGUR_RECURSION_LIMIT", DEFAULT_AUGUR_RECURSION_LIMIT)`, keeping the `if recursion_limit` would let someone intentionally suppress the default (i.e. use Python's default). This is of dubious utility, though, so I'd probably roll the default and the value test into one thing instead, something like:

```
DEFAULT_AUGUR_RECURSION_LIMIT = 10000
sys.setrecursionlimit(int(os.environ.get("AUGUR_RECURSION_LIMIT") or DEFAULT_AUGUR_RECURSION_LIMIT))
```

Reply:
Ah, good catch! Updated to the suggested change in 1834540b1a61287d242f0afc8137c507611b47fd.